bundles/org.openhab.binding.velux/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Velux Bridge Things Binding -->
+<thing:thing-descriptions bindingId="velux"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Velux Bridge Binding -->
+	<bridge-type id="klf200">
+		<label>@text/bridge-type.velux.klf200.label</label>
+		<description>@text/bridge-type.velux.klf200.description</description>
+
+		<channels>
+			<channel id="status" typeId="status" />
+			<channel id="reload" typeId="reload" />
+			<channel id="downtime" typeId="downtime" />
+			<channel id="doDetection" typeId="doDetection" />
+
+			<channel id="firmware" typeId="firmware" />
```

Comment:
@kaikreuzer I'm in doubt about the modeling here, your opinion is appreciated. The following channels look more like properties to me.

Reply:
Yes, you are absolutely right.
- For firmware, we even have a [property defined](https://github.com/openhab/openhab-core/blob/3811cb95604f69f6a3d77da2caf391d81b05458e/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/Thing.java#L55), which should be used.
- ipAddress is already a configuration parameter of the bridge, so it must not be a channel at the same time.
- All the rest is probably not really helpful at all - I do not recommend adding information just because it is there. Especially showing password might be counter-productive. IF the information is desired, it should be put into properties, because it is not expected to change or be changed by openHAB.

Reply:
Adapted.